postgres0.yml

```diff
@@ -35,6 +35,9 @@ bootstrap:
       #primary_slot_name: patroni
     postgresql:
       use_pg_rewind: true
+      # Fully qualified kerberos ticket file for the running user
```

Comment:
It should be under the parameters. And I think it would be better to move it off `bootstrap.dcs` to the `postgresql` section (it is highly unlikely but filename could different from host to host).